docs/build/tooling/hardhat/hardhat-zksync-upgradable.md

```diff
@@ -606,22 +606,20 @@ async function main() {
 
   const beaconAddress = await beacon.getAddress();
 
-  const boxBeaconProxy = await hre.zkUpgrades.deployBeaconProxy(deployer.zkWallet, beaconAddress, contract, [42]);
+  const boxBeaconProxy = await hre.zkUpgrades.deployBeaconProxy(deployer.zkWallet, beaconAddress, contract, []);
   await boxBeaconProxy.waitForDeployment();
 
   // upgrade beacon
   const boxV2Implementation = await deployer.loadArtifact("BoxV2");
   await hre.zkUpgrades.upgradeBeacon(deployer.zkWallet, beaconAddress, boxV2Implementation);
-  console.info(chalk.green("Successfully upgraded beacon Box to BoxV2 on address: ", beaconAddress));
+  console.info("Successfully upgraded beacon Box to BoxV2 on address: ", beaconAddress);
 
   const attachTo = new zk.ContractFactory<any[], Contract>(boxV2Implementation.abi, boxV2Implementation.bytecode, deployer.zkWallet, deployer.deploymentType);
   const upgradedBox = attachTo.attach(await boxBeaconProxy.getAddress());
 
   upgradedBox.connect(zkWallet);
   // wait some time before the next call
   await new Promise((resolve) => setTimeout(resolve, 2000));
-  const value = await upgradedBox.retrieve();
```

Comment:
This should also work based on the BoxV2 from this example. Can you double check that your BoxV2 has the following:

```solidity
    // Reads the last stored value and returns it with a prefix
    function retrieve() public view returns (string memory) {
        return string(abi.encodePacked("V2: ", uint2str(value)));
    }
```

Reply:
Rollbacked changes - all works as expected